source/core/src/test/com/csse3200/game/entities/factories/NPCFactoryTest.java

```diff
@@ -724,6 +728,113 @@ void TestSnakeSetPosition() {
         assertEquals(pos, snake.getPosition());
     }
 
+    /**
+<<<<<<< HEAD
+     * Tests Creation of a magpie.
+     */
+    @Test
+    void TestMagpieCreation() {
+        Assertions.assertNotNull(magpie, "magpie should not be null.");
+    }
+
+    /**
```

Comment:
Most of these tests can probably be combined into a `testInititialisation` test - I don't think you need so many tests just to test the constructor.

Reply:
all done!! thank you